tests/Doctrine/Performance/Annotations/CachedReadPerformanceWithInMemoryBench.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Performance\Annotations;
+
+use Doctrine\Annotations\AnnotationReader;
+use Doctrine\Annotations\CachedReader;
+use Doctrine\Common\Cache\ArrayCache;
+use Doctrine\Tests\Annotations\Fixtures\Controller;
+use ReflectionMethod;
+
+/**
+ * @BeforeMethods({"initialize"})
+ */
+final class CachedReadPerformanceWithInMemoryBench
+{
+    /** @var CachedReader */
+    private $reader;
+
+    /** @var ReflectionMethod */
+    private $method;
+
+    public function initialize() : void
+    {
+        $this->reader = new CachedReader(new AnnotationReader(), new ArrayCache());
```

Comment:
Should probably use a void cache in this sort of test. Also clashes with `CachedReader` removal - maybe take the chance and get rid of it overall?

Reply:
#249 😛